client/styles/components/_console.scss

```diff
@@ -102,4 +127,15 @@
 	.preview-console--collapsed & {
 		display: none;
 	}
+}
+
+.preview-console__logged-times {
+	font-size: 10px;
+	font-weight: bold;
+	margin: 2px 0 0 6px;
+	border-radius: 100px;
+	padding: 1px 4px;
+	z-index: 100;
+	float: left;
```

Comment:
I don't like to use `float` because it's annoying. You should be able to change this to `left: 0` and i think you'll get the same effect.